packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -21,12 +21,70 @@ import {
   buildParamEntity,
 } from './entity-builder';
 
-export function resolveInterfaceMembers(
```

Comment:
`resolveInterfaceMembers` wasn't actually removed, it was moved below and it's no longer exported.